tests/PHPStan/Analyser/nsrt/bug-10893.php

```diff
@@ -10,16 +10,16 @@
 function hasMicroseconds(\DateTimeInterface $value, string $str): bool
 {
 	assertType('non-falsy-string&numeric-string', $str);
-	assertType('int', (int)$str);
-	assertType('bool', (int)$str !== 0);
+	assertType('int<min, -1>|int<1, max>', (int)$str);
+	assertType('true', (int)$str !== 0);
 
 	assertType('non-falsy-string&numeric-string', $value->format('u'));
-	assertType('int', (int)$value->format('u'));
-	assertType('bool', (int)$value->format('u') !== 0);
+	assertType('int<min, -1>|int<1, max>', (int)$value->format('u'));
```

Comment:
This is obviously wrong test change - see https://github.com/phpstan/phpstan-src/pull/3980#issuecomment-2886594537